src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -246,6 +246,12 @@ ManageCLIError.BucketAlreadyExists = Object.freeze({
     http_code: 409,
 });
 
+ManageCLIError.BucketCleanUpFailed = Object.freeze({
+    code: 'BucketCleanUpFailed',
+    message: 'Bucket clean up failed',
```

Comment:
Maybe change the name and the description to capture the fact that it was for the temp dir.